vcd/resource_vcd_nsxt_edgegateway_test.go

```diff
@@ -106,6 +109,23 @@ func TestAccVcdNsxtEdgeGateway(t *testing.T) {
 	postTestChecks(t)
 }
 
+// in CDS cluster ID isn't accessible: Forbidden: User is not authorized to perform this operation on the application. Please contact the system administrator to get access., error code 401
+func ifPossibleAddClusterId(t *testing.T, err error, vcdClient *VCDClient, params StringMap) {
```

Comment:
Please add a comment explaining what this function does. The current comment is not helpful

Reply:
@Didainius could you rephrase this one?

Reply:
added